composer.json

```diff
@@ -5,10 +5,11 @@
     "license": "MIT",
     "require": {
         "php": ">=5.4",
-        "guzzle/http": ">=3.6.0,<4"
+        "guzzlehttp/guzzle": "~6.3.3",
+        "ext-json": "*"
```

Comment:
Even though ext-json is present by default, it is apparently best practice to explicitly list it as a requirement if you use things like `json_decode`.